### PR TITLE
Add local mode to ftl

### PR DESCRIPTION
**Pull request by priyawadhwa**

### CODE REVIEW

File: ftl/node/main.py [hunk 1]
```diff
@@ -96,6 +108,18 @@ def main(args):
             logging.info('Generating app layer...')
             app_layer, diff_id = bldr.BuildAppLayer()
             with append.Layer(deps, app_layer, diff_id=diff_id) as app_image:
+                if args.local:
+                    if not args.filepath:
+                        raise AssertionError('Please provide path to tar \
+                                             with --filepath flag')
+                    dir = os.path.dirname(args.filepath)
```
Comment (dlorenc) [inline ftl/node/main.py]: You could leave off this check and just assume the user created a tar ahead of time.

Comment (dlorenc) [reply ftl/node/main.py]: We might also want to swap out or disable the cache if --local mode is used. WDYT?

Comment (priyawadhwa) [reply ftl/node/main.py]: I think disabling it could make sense, but what would we swap it out for?

**Comment by priyawadhwa** [comment]
Hey @dlorenc, do we actually need the --local flag? It could be implied we want to save it locally if the --filepath is provided

**Comment by dlorenc** [comment]
> Hey @dlorenc, do we actually need the --local flag? It could be implied we want to save it locally if the --filepath is provided

Nope! Good call.

File: ftl/node/main.py [hunk 2]
```diff
@@ -56,6 +57,12 @@
     help='The path where the application data sits.')
 
 parser.add_argument(
+    '--filepath',
```
Comment (dlorenc) [inline ftl/node/main.py]: Nit: Something like --output-path might be a little more clear.

**Pull request merged**